pkgs/by-name/wa/watchyourlan/package.nix

```
@@ -0,0 +1,24 @@
+{ pkgs, lib, ... }:
+let
+  inherit (pkgs) buildGoModule fetchFromGitHub;
+in
```

```suggestion
{ buildGoModule, fetchFromGitHub, lib }:
```

Never pass `pkgs` or use `...` in a package expression